playlist.go

```diff
@@ -734,3 +757,65 @@ func populate(root *tview.TreeNode, rootPath string) error {
 
 	return nil
 }
+
+func (p *Playlist) yank() error {
+	yankFile = p.getCurrentFile()
+	isYanked = true
+	return nil
+}
+
+func (p *Playlist) paste() error {
```

Comment:
Maybe this requires a little bit clean up as this method won't ever return as all errors are logged and does not return the error. Return the error first and log the error once it's on command level in order to improve the log trace, use `tracerr.Wrap` before return. Exception for goroutine, all errors are logged immediately.